test-data/unit/check-class-namedtuple.test

```diff
@@ -407,7 +407,7 @@ class HasNone(NamedTuple):
     x: int
     y: Optional[int] = None
 
-reveal_type(HasNone(1))  # E: Revealed type is 'Tuple[builtins.int, builtins.int, fallback=__main__.HasNone]'
+reveal_type(HasNone(1))  # E: Revealed type is 'Tuple[builtins.int, Union[builtins.int, builtins.None], fallback=__main__.HasNone]'
```

Comment:
Off-topic: I so wish that `Union[X, None]` were rendered as `Optional[X]`... (Hm, somewhere below there *is* a test that shows that: test-data/unit/check-isinstance.test -- what's the difference?)